charts/fluent-bit/templates/_pod.tpl

```diff
@@ -13,6 +13,10 @@ securityContext:
 dnsConfig:
   {{- toYaml . | nindent 2 }}
 {{- end }}
+{{ if .Values.initContainers }}
+initContainers:
+  {{ toYaml .Values.initContainers | nindent 2 }}
```

Comment:
Please include a - at beginning to eliminate white space before new line indentation